lib/fizzy/constexpr_vector.hpp

```diff
@@ -26,7 +26,7 @@ class constexpr_vector
     {}
 
     constexpr const T* data() const noexcept { return m_array; }
```

Comment:
Shouldn't we just mark all getters as nodiscard?

Reply:
It is better to remove any current usage of `[[nodiscard]]`. This is just noise.

Reply:
Why? It supposed to warn if we don't use the result of those?

Reply:
Yes. But this may be overpedantic. Even in C++20 they only marked `empty()` functions with it so far.